README.md

```diff
@@ -43,47 +43,44 @@ DTOs, and populate services.  the PopulateCliApp can be used to populate specifi
 ### Setup
 
 #### Database setup
-run `./local-dev/run_postgres.sh start`
-This will start a postgres container with a schema and user configured
+Run `./local-dev/run_postgres.sh start`
+This will start a Postgres container with a schema and database user configured. This Postgres container is utilized by both the
+local development environment and the unit/integration tests.
 
 #### IDE Setup
 In IntelliJ, File -> New -> Protject from Existing Sources.  When importing the project, make sure it's set as a gradle project  
 ![image](https://github.com/broadinstitute/juniper/assets/2800795/dd2cf363-f761-47bc-9620-28e47a20feff)
 
 
-* **server:**
+* **Server:**
 
    * Make sure IntelliJ is set to Java 17 in *two* places
-
       * Project Structure > Project Settings > Project > SDK
       * Preferences > Build, Execution, Deployment > Build Tools > Gradle > Gradle Projects > \[this project\] > Gradle JVM
          * Recommended setting for this is "Project SDK"
    * In Preferences > Build, Execution, Deployment > Compiler > Annotation Processors, make sure annotation processing is enabled (otherwise lombok getters/setters won't work)
-   * Create two Spring Boot Run/Debug Configurations.
+   * Create these two Spring Boot Run/Debug Configurations by going to Run > Edit Configurations > + > Spring Boot
      * ApiAdminApp (in api-admin module)
-       * set active profiles of "human-readable-logging" and "development"
-       * disable launch optimization
-       * set environment variable: `REDIRECT_ALL_EMAILS_TO=<<your email address>>`
-       * set environment variable: `TDR_SA_CREDS=<<vault read -field=sa-key.json.b64 secret/dsp/ddp/d2p/dev/d2p-tdr-sa>>`
-       * set environment variable: `B2C_TENANT_NAME=ddpdevb2c`
-       * set environment variable: `B2C_CLIENT_ID=<<vault read -field value secret/dsp/ddp/b2c/dev/application_id>>`
-       * set environment variable: `B2C_POLICY_NAME=B2C_1A_ddp_participant_signup_signin_dev`
-       * set environment variable: `DSM_JWT_SIGNING_SECRET=<<vault read -field jwt_signing_secret secret/dsp/ddp/d2p/dev/dsm>>`
-       * set environment variable: `SENDGRID_API_KEY=<<vault read -field=api_key secret/dsp/ddp/d2p/dev/sendgrid>>`
+       * Set the Active profiles field to: `human-readable-logging, development`
+       * Disable launch optimization by clicking `Modify options > Disable launch optimization`
+       * Render environment variables with `./local-dev/render_environment_vars.sh ApiAdminApp <YOUR_EMAIL>`
```

Comment:
```suggestion
       * Render environment variables with `bash ./local-dev/render_environment_vars.sh ApiAdminApp <YOUR_EMAIL>`
```
The script doesn't work in zsh (and unless it's super-trivial, I don't think it's worth updating it to support zsh)

Reply:
Good call, done